aviary/docs/examples/additional_flight_phases.ipynb

```diff
@@ -72,6 +72,7 @@
     "            'duration_bounds': ((25.5, 76.5), 'min'),\n",
     "        },\n",
     "        'initial_guesses': {'time': ([0, 51], 'min')},\n",
+    "        \"initial_guesses\": {\"times\": ([0, 51], \"min\")},\n",
```

Comment:
Our `Ruff` codestyle enforces single-quotes for dictionaries, so you probably want to change these all back.